### PR TITLE
🏃🏻‍♀️ Make gerred approver

### PR DESCRIPTION
**Pull request by gerred**
Been active in reviews, very invested in this project and happy to help continue guiding CR to be an amazing tool!

### CODE REVIEW

**Comment by gerred** [comment]
My commit grammar somehow was awful, but here we are. 🤷🏻‍♂️

**Comment by DirectXMan12** [comment]
s/approver/admin (slightly different IIRC, we can discuss that though), otherwise LGTM

**Comment by DirectXMan12** [comment]
As stated above, I think this seems reasonable, but just in the spirit of fairness:

if anyone wants to burst into the room dramatically and shout: "stop, @gerred is actually an army time-travelling hyper-intelligent squirrels bent on global domination!", now is the time to do so :-)

**Comment by alexeldeib** [comment]
>  "stop, @gerred is actually an army time-travelling hyper-intelligent squirrels bent on global domination!"

I read this as as a pro, not a con 😅

**Comment by gerred** [comment]
I need this as an avatar now @DirectXMan12. :shipit:

**Comment by gerred** [comment]
updated PR title to approver

**Comment by kensipe** [comment]
+1 NB release the time traveling squirrels!

**Comment by alenkacz** [comment]
woohoo, go @gerred 👍

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -5,11 +5,11 @@ aliases:
   - directxman12
   - droot
   - pwittrock
+  - gerred
```
Comment (DirectXMan12) [inline OWNERS_ALIASES]: please add an `approvers` section, otherwise, LGTM

**Comment by gerred** [comment]
@DirectXMan12 ah looks like maintainers is the right section here, fixing now

**Comment by gerred** [comment]
@DirectXMan12 done!

**Comment by vincepri** [comment]
/lgtm

@mengqiy @DirectXMan12 Given that it has been 7 days for lazy approval, I think this should be good to go?

**Comment by gerred** [comment]
@DirectXMan12 to resolve this, should I put myself under maintainer or approver?

**Comment by DirectXMan12** [comment]
I'll fix it.

**Pull request merged**